internal/query/map.go

```diff
@@ -1,56 +1,117 @@
 package query
 
 import (
+	"errors"
 	"fmt"
 	"strings"
 )
 
 // GetMap returns a map, which satisfies conditions.
-func GetMap(query map[string][]string, key string) (dicts map[string]interface{}, exists bool) {
+func GetMap(query map[string][]string, key string) (dicts map[string]interface{}, err error) {
 	result := make(map[string]interface{})
+	getAll := key == ""
+	var allErrors = make([]error, 0)
```

Comment:
Consider `errors.Join` which is available since `go 1.20`

Reply:
I'm using it at the end (in return), 
but I don't know how to use it in the middle, errors.Join is a little bit not planned the best, 
if we will make errors.Join each time we will end up with something like this:
```json
{
  "errs": [
    "message 1",
    {
      "errs": [
        "message 2",
        {
          "errs": [
            "message 3"
          ]
        }
      ]
    }
  ]
}
```